kms-message/src/kms_kmip_tag_type_private.h

```diff
@@ -0,0 +1,319 @@
+   KMIP_TAG_ValidationAuthorityURI = 0x4200E2,
+   KMIP_TAG_ValidationVersionMajor = 0x4200E3,
+   KMIP_TAG_ValidationVersionMinor = 0x4200E4,
+   KMIP_TAG_ValidationType = 0x4200E5,
+   KMIP_TAG_ValidationLevel = 0x4200E6,
+   KMIP_TAG_ValidationCertificateIdentifier = 0x4200E7,
+   KMIP_TAG_ValidationCertificateURI = 0x4200E8,
+   KMIP_TAG_ValidationVendorURI = 0x4200E9,
+   KMIP_TAG_ValidationProfile = 0x4200EA,
+   KMIP_TAG_ProfileInformation = 0x4200EB,
+   KMIP_TAG_ProfileName = 0x4200EC,
+   KMIP_TAG_ServerURI = 0x4200ED,
+   KMIP_TAG_ServerPort = 0x4200EE,
+   KMIP_TAG_StreamingCapability = 0x4200EF,
+   KMIP_TAG_AsynchronousCapability = 0x4200F0,
+   KMIP_TAG_AttestationCapability = 0x4200F1,
+   KMIP_TAG_UnwrapMode = 0x4200F2,
+   KMIP_TAG_DestroyAction = 0x4200F3,
+   KMIP_TAG_ShreddingAlgorithm = 0x4200F4,
+   KMIP_TAG_RNGMode = 0x4200F5,
+   KMIP_TAG_ClientRegistrationMethod = 0x4200F6,
+   KMIP_TAG_CapabilityInformation = 0x4200F7,
+   KMIP_TAG_KeyWrapType = 0x4200F8,
+   KMIP_TAG_BatchUndoCapability = 0x4200F9,
+   KMIP_TAG_BatchContinueCapability = 0x4200FA,
+   KMIP_TAG_PKCS12FriendlyName = 0x4200FB,
+   KMIP_TAG_Description = 0x4200FC,
+   KMIP_TAG_Comment = 0x4200FD,
+   KMIP_TAG_AuthenticatedEncryptionAdditionalData = 0x4200FE,
+   KMIP_TAG_AuthenticatedEncryptionTag = 0x4200FF,
+   KMIP_TAG_SaltLength = 0x420100,
+   KMIP_TAG_MaskGenerator = 0x420101,
+   KMIP_TAG_MaskGeneratorHashingAlgorithm = 0x420102,
+   KMIP_TAG_PSource = 0x420103,
+   KMIP_TAG_TrailerField = 0x420104,
+   KMIP_TAG_ClientCorrelationValue = 0x420105,
+   KMIP_TAG_ServerCorrelationValue = 0x420106,
+   KMIP_TAG_DigestedData = 0x420107,
+   KMIP_TAG_CertificateSubjectCN = 0x420108,
+   KMIP_TAG_CertificateSubjectO = 0x420109,
+   KMIP_TAG_CertificateSubjectOU = 0x42010A,
+   KMIP_TAG_CertificateSubjectEmail = 0x42010B,
+   KMIP_TAG_CertificateSubjectC = 0x42010C,
+   KMIP_TAG_CertificateSubjectST = 0x42010D,
+   KMIP_TAG_CertificateSubjectL = 0x42010E,
+   KMIP_TAG_CertificateSubjectUID = 0x42010F,
+   KMIP_TAG_CertificateSubjectSerialNumber = 0x420110,
+   KMIP_TAG_CertificateSubjectTitle = 0x420111,
+   KMIP_TAG_CertificateSubjectDC = 0x420112,
+   KMIP_TAG_CertificateSubjectDNQualifier = 0x420113,
+   KMIP_TAG_CertificateIssuerCN = 0x420114,
+   KMIP_TAG_CertificateIssuerO = 0x420115,
+   KMIP_TAG_CertificateIssuerOU = 0x420116,
+   KMIP_TAG_CertificateIssuerEmail = 0x420117,
+   KMIP_TAG_CertificateIssuerC = 0x420118,
+   KMIP_TAG_CertificateIssuerST = 0x420119,
+   KMIP_TAG_CertificateIssuerL = 0x42011A,
+   KMIP_TAG_CertificateIssuerUID = 0x42011B,
+   KMIP_TAG_CertificateIssuerSerialNumber = 0x42011C,
+   KMIP_TAG_CertificateIssuerTitle = 0x42011D,
+   KMIP_TAG_CertificateIssuerDC = 0x42011E,
+   KMIP_TAG_CertificateIssuerDNQualifier = 0x42011F,
+   KMIP_TAG_Sensitive = 0x420120,
+   KMIP_TAG_AlwaysSensitive = 0x420121,
+   KMIP_TAG_Extractable = 0x420122,
+   KMIP_TAG_NeverExtractable = 0x420123,
+   KMIP_TAG_ReplaceExisting = 0x420124,
```

Comment:
If we want to remove trailing commas in enumerator lists, there is one more here.

Reply:
Good catch, done.